Kitodo-DataManagement/src/main/resources/db/migration/V2_134__Add_import_configuration_id_to_process_and_permission_to_update_metadata.sql

```diff
@@ -0,0 +1,21 @@
+--
+-- (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+--
+-- This file is part of the Kitodo project.
+--
+-- It is licensed under GNU General Public License version 3 or later.
+--
+-- For the full copyright and license information, please read the
+-- GPL3-License.txt file that was distributed with this source code.
+--
+
+SET SQL_SAFE_UPDATES = 0;
+
+-- add column "import_configuration_id" to "process" table
+ALTER TABLE process ADD import_configuration_id INT(11);
```

Comment:
Not sure if it makes sense to only change it here, but just to note: I get a deprecation warning here

> DB: Integer display width is deprecated and will be removed in a future release. (SQL State: HY000 - Error Code: 1681)

The new recommended way to declare integers seems to be:

ALTER TABLE process ADD import_configuration_id INT;

See for Mysql:

https://dev.mysql.com/doc/refman/8.0/en/numeric-type-attributes.html

> As of MySQL 8.0.17, the ZEROFILL attribute is deprecated for numeric data types, as is the display width attribute for integer data types. You should expect support for ZEROFILL and display widths for integer data types to be removed in a future version of MySQL. Consider using an alternative means of producing the effect of these attributes. For example, applications can use the [LPAD()](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_lpad) function to zero-pad numbers up to the desired width, or they can store the formatted numbers in [CHAR](https://dev.mysql.com/doc/refman/8.0/en/char.html) columns.

Reply:
> ALTER TABLE process ADD import_configuration_id INT;

Done.